lib/money-rails/active_record/migration_extensions/table_pg_rails4.rb

```diff
@@ -2,16 +2,16 @@ module MoneyRails
   module ActiveRecord
     module MigrationExtensions
       module Table
-        def monetize(accessor, options={})
+        def monetize(accessor, options = {})
           [:amount, :currency].each do |attribute|
-            column_present, _, *opts = OptionsExtractor.extract attribute, :no_table, accessor, options
-            column(*opts) if column_present
+            column_present, _, column_name, column_type, opts = OptionsExtractor.extract attribute, :no_table, accessor, options
+            column(column_name, column_type, **opts) if column_present
           end
         end
 
-        def remove_monetize(accessor, options={})
+        def remove_monetize(accessor, options = {})
           [:amount, :currency].each do |attribute|
-            column_present, _, column_name, _, _ =  OptionsExtractor.extract attribute, :no_table, accessor, options
+            column_present, _, column_name, = OptionsExtractor.extract attribute, :no_table, accessor, options
```

Comment:
I'm assuming that comma after `column_name` is not needed ^

Reply:
its needed. it throws away the other return values. instead of making an array of them.

Reply:
Does it?

```ruby
>> a, b, c = [1,2,3,4]
=> [1, 2, 3, 4]
>> p a, b, c
1
2
3
````

Reply:
yup, your right, my bad.